--- plugins/org.eclipse.pde.build/templates/package-build/prepare-build-dir.sh.orig	2010-09-22 18:11:55.546685423 -0400
+++ plugins/org.eclipse.pde.build/templates/package-build/prepare-build-dir.sh	2010-09-22 17:58:40.763674792 -0400
@@ -57,7 +57,7 @@
     fi
 
     if [ "x$ERROR" != "xyes" ]; then
-      if [ $TESTING != true -o `echo $PROJECTNAME | grep "org.eclipse"` ]; then
+      if [ $TESTING != true ] || `echo "$PROJECTNAME" | grep "org.eclipse"`; then
         echo "  making symlink: $BUILDDIR/features/$PROJECTNAME -> $PROJECTDIR"
         ln -sfT "$PROJECTDIR" $BUILDDIR/features/"$PROJECTNAME"
       fi
@@ -91,7 +91,7 @@
     fi
 
     if [ "x$ERROR" != "xyes" ]; then
-      if [ $TESTING != true -o `echo $PROJECTNAME | grep "org.eclipse"` ]; then
+      if [ $TESTING != true ] || `echo $PROJECTNAME | grep "org.eclipse"`; then
         echo "  making symlink: $BUILDDIR/plugins/$PROJECTNAME -> $dir"
         ln -sfT "$dir" $BUILDDIR/plugins/"$PROJECTNAME"
       fi
